networkx/algorithms/dag.py

```diff
@@ -283,6 +307,12 @@ def lexicographical_topological_sort(G, key=None):
     RuntimeError
         If `G` is changed while the returned iterator is being processed.
 
+    Examples
+    --------
+    >>> DG = nx.DiGraph([(2, 3), (1, 3)])
+    >>> list(nx.lexicographical_topological_sort(DG))
+    [1, 2, 3]
+
```

Comment:
This is a good example to start with, but it also might be nice to add an example using the `key` kwarg to illustrate how `lexicographical_topological_sort` differs from `topological_sort`.